src/app/public/modules/flyout/types/flyout-action.ts

```diff
@@ -1,3 +1,6 @@
+/**
+ * @internal
```

Comment:
I'm pretty sure this is a public-facing feature that needs to be added to the docs (even if we don't have a description yet)?

Reply:
Yep. Done.